From e5036f2c749961a0da9913be357b7b0b267535a5 Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@freescale.com>
Date: Fri, 5 Feb 2016 15:17:14 +0200
Subject: [PATCH 053/245] fman: update an outdated header

mpc85xx.h has been moved to include/linux/fsl/svr.h

Signed-off-by: Camelia Groza <camelia.groza@freescale.com>
---
 drivers/net/ethernet/freescale/sdk_fman/Peripherals/FM/fm.c            | 2 +-
 drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm_port.c   | 2 +-
 drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_ioctls_fm.c | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/freescale/sdk_fman/Peripherals/FM/fm.c b/drivers/net/ethernet/freescale/sdk_fman/Peripherals/FM/fm.c
index 7b9f084..71ef3a2 100644
--- a/drivers/net/ethernet/freescale/sdk_fman/Peripherals/FM/fm.c
+++ b/drivers/net/ethernet/freescale/sdk_fman/Peripherals/FM/fm.c
@@ -47,7 +47,7 @@
 #include "fm_common.h"
 #include "fm_ipc.h"
 #include "fm.h"
-#include <asm/mpc85xx.h>
+#include <linux/fsl/svr.h>
 #include "fsl_fman.h"
 
 
diff --git a/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm_port.c b/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm_port.c
index fa73867..2cba7a5 100644
--- a/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm_port.c
+++ b/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm_port.c
@@ -51,7 +51,7 @@
 #include <linux/cdev.h>
 #include <linux/slab.h>
 #include <linux/spinlock.h>
-#include <asm/mpc85xx.h>
+#include <linux/fsl/svr.h>
 
 #include "sprint_ext.h"
 #include "fm_common.h"
diff --git a/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_ioctls_fm.c b/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_ioctls_fm.c
index fcc0198..2476dfa 100644
--- a/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_ioctls_fm.c
+++ b/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_ioctls_fm.c
@@ -60,7 +60,7 @@
 #include <asm/uaccess.h>
 #include <asm/errno.h>
 #include <sysdev/fsl_soc.h>
-#include <asm/mpc85xx.h>
+#include <linux/fsl/svr.h>
 
 #if defined(CONFIG_COMPAT)
 #include <linux/compat.h>
-- 
2.1.0.27.g96db324

